src/web/app/pages-static/static-pages.module.ts

```diff
@@ -68,12 +68,18 @@ const routes: Routes = [
       {
         path: 'getting-started',
         component: InstructorHelpGettingStartedComponent,
+        data: {
+          instructorHelpPath: 'web/front/help/instructor',
+        },
       },
     ],
   },
   {
     path: 'getting-started',
```

Comment:
Come to think about it, do you need this routing, i.e. is `/web/front/getting-started` referenced anywhere?

Reply:
Is this supposed to be referenced at the nav bar? I realized currently this routing isn't referenced anywhere so it's quite redundant

Reply:
If this routing is not used in the first place, let's just remove it.

Reply:
Done! `/web/front/getting-started` is now invalid